articles/post.php

```diff
@@ -2,6 +2,13 @@
 $post_share_url = esc_url( get_permalink() );
 $post_share_title = rawurlencode( spine_get_title() );
 $post_share_placement = spine_get_option( 'post_social_placement' );
+$spine_social_options = spine_social_options();
+if ( ! empty( $spine_social_options['twitter'] ) ) {
+	$twitter_array = array_filter( explode( '/', $spine_social_options['twitter'] ) );
```

Comment:
I don't think the `array_filter()` is necessary here. We should be able to do `$twitter_array = explode( '/', $spine....`